ev-dev-tools/src/ev_cli/templates/interface-Base.hpp.j2

```diff
@@ -7,7 +7,7 @@
 
 #include <framework/ModuleAdapter.hpp>
 #include <utils/types.hpp>
-#include <utils/error.hpp>
+#include <utils/error/error.hpp>
```

Comment:
This would necessitate a bump in the version in ev-cli (and the reference to it in everest-core)

Reply:
As discussed: Move this PR to december release